modules/openshift-cluster-maximums-environment.adoc

```diff
@@ -129,4 +129,4 @@
 3. No separate workload node was used. The workload simulates a micro-service workload between two compute nodes.
 4. Physical number of processors used is six Integrated Facilities for Linux (IFLs).
 5. Total physical memory used is 512 GiB.
```

Comment:
- "micro-service" should be "microservice"
- I'm assuming that GiB is correct here instead of GB